dask/dataframe/io/io.py

```diff
@@ -562,12 +565,19 @@ def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed"):
 
     name = prefix + "-" + tokenize(*dfs)
     dsk = merge(df.dask for df in dfs)
-    dsk.update(
-        {
-            (name, i): (check_meta, df.key, meta, "from_delayed")
-            for (i, df) in enumerate(dfs)
-        }
-    )
+    if verify_meta:
+        dsk.update(
+            {
+                (name, i): (check_meta, df.key, meta, "from_delayed")
+                for (i, df) in enumerate(dfs)
+            }
+        )
+    else:
+        dsk.update(
+            {
+                (name, i): (df.key,) for (i, df) in enumerate(dfs)
```

Comment:
I suspect that you just want `df.key` not `(df.key,)`.  Here is the spec if you're interested: https://docs.dask.org/en/latest/spec.html